INSTALL

```diff
@@ -109,7 +109,7 @@ Prepare your system
  2. Install dtrace (to generate provider.h)
     sudo apt-get install systemtap-sdt-dev
  3. Install libdispatch pre-reqs
-    sudo apt-get install libblocksruntime-dev libkqueue-dev libbsd-dev
+    sudo apt-get install libblocksruntime-dev libbsd-dev
```

Comment:
I know, but the situation with libpwq is different: libdispatch is its only client on the system *ever* so having dispatch "disptribute" it is fine.

libkqueue is different, it has other clients on the system and other OSS projects use it, so while I understand the motivation of this patch for swift development purposes, I don't think we can ship like that.